blueprints/tools/blueprint_file_converter.py

```diff
@@ -31,98 +30,92 @@ def file_converter_versions():
 @file_converter_routes.route("/allowed_files", methods=["GET"])
 def file_converter_allowed_files():
     extensions = geode_functions.list_all_input_extensions()
-    return {"status": 200, "extensions": extensions}
+    return flask.jsonify(status=200, extensions=extensions)
 
 
 @file_converter_routes.route("/allowed_objects", methods=["POST"])
 def file_converter_allowed_objects():
-    array_variables = ["filename"]
-    variables_dict = geode_functions.get_form_variables(
-        flask.request.form, array_variables
-    )
-    file_extension = os.path.splitext(variables_dict["filename"])[1][1:]
-    allowed_objects = geode_functions.list_objects(file_extension)
-
-    return flask.make_response({"allowed_objects": allowed_objects}, 200)
+    data = flask.request.json
+    filename = data.get("filename")
```

Comment:
By doing that you're remooving the automatic error gestion.
You should keep with the 'get_form_variables' and we should make some changes to it if necessary

Reply:
Ok !